.travis.yml

```diff
@@ -2,9 +2,8 @@ language: php
 
 php:
   - hhvm
+  - 7.0
```

Comment:
You're just stuck using older versions of the dependencies too which have also moved on and dropped support for dead versions of PHP.